docs/V4_API_Updates.md

```diff
@@ -32,6 +32,19 @@ You must manually add the autogenerated file we committed to the repository
 to your project, and `#import RevenueCat-Swift.h` in your bridging header. You can see how we do this in our
 [SPMIntegration project](https://github.com/RevenueCat/purchases-ios/tree/main/IntegrationTests/SPMIntegration).
 
+### StoreKit 2
```

Comment:
Not sure if this is the best place. Maybe `README.md` instead?

Reply:
I like having it here since it is an API change. Wouldn't hurt to _also_ include a note in `README.md`, though